dashboard/src/App.js

```diff
@@ -16,11 +17,14 @@ const PodInfoView = ({pod, namespace}) => (
   </Segment>
 );
 
-const OperatorsView = ({error, deployment, pod, namespace}) => {
+const OperatorsView = ({error, deployment, storage, pod, namespace}) => {
   const podInfoView = (<PodInfoView pod={pod} namespace={namespace}/>);
   if (deployment) {
     return (<DeploymentOperator podInfoView={podInfoView} error={error}/>);
   }
+  if (storage) {
+    return (<StorageOperator podInfoView={podInfoView} error={error}/>);
+  }
```

Comment:
If you want to avoid repeating yourself a bit, you could do something like this:

```js
let Operator = NoOperator;
if (deployment) Operator = DeploymentOperator;
else if (storage) Operator = StorageOperator;
return (
  <Operator
    podInfoView={
      <PodInfoView pod={pod} namespace={namespace} />
    }
    error={error}
  />
);
```

Reply:
changed